scripts/timeseries_comparison/comparison.py

```diff
@@ -0,0 +1,154 @@
+"""
+Compare G.h data with WHO timeseries
+"""
+import io
+import os
+import json
+import datetime
+import logging
+
+import boto3
+import pandas as pd
+import pycountry
+
+s3 = boto3.resource("s3")
+
+UK_COUNTRIES = ["England", "Wales", "Scotland", "Northern Ireland"]
+
+COUNTRY_ISO3_QUIRKS = {
+    "democratic republic of the congo": "COD",
+    "iran": "IRN",
+    "republic of congo": "COG",
+    "russia": "RUS",
+}
+
+def require_env(vars: list[str]):
+    "Returns required environment variables in a dictionary or aborts"
+    env = {v: os.getenv(v) for v in vars}
+    if absent_vars := [k for k, v in env.items() if v is None]:
+        raise ValueError(f"No value specified for {', '.join(absent_vars)}")
+    return env
```

Comment:
No need to pass env around, it's global.

Reply:
I did this so that main() can be called and tested without access to env

Reply:
Seems better to inject different values in a test environment.